controllers/grid/settings/reviewForms/ReviewFormGridCellProvider.inc.php

```diff
@@ -29,24 +29,15 @@ function getTemplateVarsFromRowColumn($row, $column) {
 		assert(is_a($element, 'ReviewForm') && !empty($columnId));
 		switch ($columnId) {
 			case 'name':
-				$label = $element->getLocalizedTitle();
-				return array('label' => $label);
-				break;
+				return array('label' => $element->getLocalizedTitle());
 			case 'inReview':
-				$label = $element->getIncompleteCount();
-				return array('label' => $label);
-				break;
+				return array('label' => $element->getIncompleteCount());
 			case 'completed':
-				$label = $element->getCompleteCount();
-				return array('label' => $label);
-				break;
+				return array('label' => $element->getCompleteCount());
 			case 'active':
-				$selected = $element->getActive();
-				return array('selected' => $selected);
-				break;
-			default:
-				break;
+				return array('selected' => $element->getActive());
 		}
+		return parent::getTemplateVarsFromRowColumn($row, $column);
```

Comment:
This is a minor nitpick, but if this were ever sub-classed, or if we ever wanted `GridCellProvider::getTemplateVarsFromRowColumn` to return a global configuration parameter, the parent method would get skipped with the `returns` in the `switch`.

My preference would be to merge the results of the parent with the child in cases like this:

```
switch ($columnId) {
  case 'name':
    $vars = array('label' => 'whatever');
  ...
}
$parentVars = parent::getTemplateVarsFromRowColumns($row, $column);
return array_merge($parentVars, $vars);
```

Reply:
That is a bigger convention change, but it's a good idea. However, best not dealt with in this context. I have some plans for heavier lifting in parent classes etc. and that would be a good time to introduce this.